docs/user/tutorial/60-admin.rst

```diff
@@ -44,12 +40,33 @@ Add an image to the Filer
 
 .. todo:: write up steps properly
 
+.. image:: images/new_folder_button.png
```

Comment:
Also if you can, please take screenshots on a retina display for better resolution.